did-component.html

```diff
@@ -1,8 +1,12 @@
 <div class="well">
   <div class="text-center">
-    <button type="button" class="btn btn-primary btn-lg"
-      ng-disabled="$ctrl.loading"
-      ng-click="$ctrl.login()"><i class="fa fa-cloud"></i>
-        Sign in with Decentralized Identity</button>
+    <button type="button" class="btn btn-primary btn-lg hidden-sm hidden-xs"
+      ng-disabled="$ctrl.loading" ng-click="$ctrl.login()">
+      <i class="fa fa-cloud"></i> Sign in with Decentralized Identity
+    </button>
+    <button type="button" class="btn btn-primary hidden-md hidden-lg"
+      ng-disabled="$ctrl.loading" ng-click="$ctrl.login()">
+      <i class="fa fa-cloud"></i> Sign in with DID
```

Comment:
I think we should make just one of these and make it simply say "Sign in" and change the icon to be `fa-id-card`. Hopefully that will be enough differentiation if there are other options shown.